internal/api/deployments.go

```diff
@@ -75,3 +82,22 @@ type DeploymentUpdate struct {
 	WorkPoolName           string                 `json:"work_pool_name,omitempty"`
 	WorkQueueName          string                 `json:"work_queue_name,omitempty"`
 }
+
+// ConcurrencyOptions is a representation of the deployment concurrency options.
+type ConcurrencyOptions struct {
+	CollisionStrategy string `json:"collision_strategy"`
+}
+
+// GlobalConcurrencyLimit is a representation of the deployment global concurrency limit.
+type GlobalConcurrencyLimit struct {
+	Limit int `json:"limit"`
```

Comment:
should we just make this an int64, since you're casting it

Reply:
good catch, updated in 599e7e0